app/controllers/publishers/vacancies/job_applications_controller.rb

```diff
@@ -31,6 +31,20 @@ def download_pdf
     )
   end
 
+  def download_application_form
+    unless @job_application.application_form.attached?
+      redirect_to organisation_job_job_application_path(vacancy.id, @job_application.id), alert: I18n.t("publishers.vacancies.job_applications.download_pdf.no_file")
+      return
+    end
+
+    send_data(
+      @job_application.application_form.download,
+      filename: @job_application.application_form.filename.to_s,
+      type: @job_application.application_form.content_type,
+      disposition: "inline",
+    )
+  end
```

Comment:
we could potentially have just one `download`  endpoint for both type of job application.